.flake8

```diff
@@ -0,0 +1,25 @@
+[flake8]
+# We disable the following inspections:
+# 1. F541: f-string is missing placeholders (we allow f-strings that don't do
+#    any formatting for consistent looks and for future safety)
+# 2. Inspections incompatible with Black (see https://github.com/psf/black/blob/master/docs/compatible_configs.md#why-those-options-above-1):
+#    E203: whitespace before ':'
+# 3. Q003 ("Change outer quotes to avoid escaping inner quotes")
+# 4. E501 "line too long" (covered by black)
+# 6. W291 (trailing-whitespace)
+# 7. W293 (blank line contains whitespace)
+# 8. E227 (missing whitespace around bitwise or shift operator)
+# 9. E125 (continuation line with same indent as next logical line - covered by PyCharm)
+# 10. F401 imported but unused (covered by PyCharm)
+extend-ignore: F541,E203,Q003,E501,W291,W293,E227,E125,F401
+
+# Allow listing module imports in __init__.py
```

Comment:
I'm just curious, is that not allowed by default?

Reply:
It would report them as unused imports

Reply:
Alternatively I can add `__all__ = ['Pipeline', ...]` into `__init__.py` (see https://stackoverflow.com/questions/31079047/python-pep8-class-in-init-imported-but-not-used), but it's duplication